ulib/experimental/Steel.Effect.fsti

```diff
@@ -374,33 +209,48 @@ let bind_pure_steel__ens (#a:Type) (#b:Type)
 
 val bind_pure_steel_ (a:Type) (b:Type)
   (#[@@@ framing_implicit] wp:pure_wp a)
+  (#framed:eqtype_as_type bool)
   (#[@@@ framing_implicit] pre:pre_t) (#[@@@ framing_implicit] post:post_t b)
   (#[@@@ framing_implicit] req:a -> req_t pre) (#[@@@ framing_implicit] ens:a -> ens_t pre b post)
-  (f:eqtype_as_type unit -> PURE a wp) (g:(x:a -> repr b pre post (req x) (ens x)))
+  (f:eqtype_as_type unit -> PURE a wp) (g:(x:a -> repr b framed pre post (req x) (ens x)))
 : repr b
+    framed
     pre
     post
     (bind_pure_steel__req wp req)
     (bind_pure_steel__ens wp ens)
 
-polymonadic_bind (PURE, SteelF) |> SteelF = bind_pure_steel_
+polymonadic_bind (PURE, SteelBase) |> SteelBase = bind_pure_steel_
 
-polymonadic_bind (PURE, Steel) |> Steel = bind_pure_steel_
+// AF: The lift below works, but the framing tactic currently requires annotations
```

Comment:
May be clean it up from here and add a separate note somewhere in wiki or github or fstar-priv?

Reply:
Cleaned up